### PR TITLE
Updating travis settings to be more standard

### PR DESCRIPTION
**Pull request by dhensby**
beep boop - I'm not a robot, I did this by hand...

### CODE REVIEW

**Comment by wilr** [comment]
Do we need to test 5.3?

**Comment by tractorcow** [comment]
Yes. ;D

**Comment by dhensby** [comment]
I've re-run the tests and I'm seeing green ticks :)

File: .travis.yml [hunk 1]
```diff
@@ -1,26 +1,33 @@
...
+    - php: 5.6
+      env: DB=MYSQL CORE_RELEASE=3.3
 
 before_script:
- - git clone git://github.com/silverstripe-labs/silverstripe-travis-support.git ~/travis-support
- - php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss --require silverstripe/cms
- - cd ~/builds/ss
+  - composer self-update || true
+  - git clone git://github.com/silverstripe-labs/silverstripe-travis-support.git ~/travis-support
+  - php ~/travis-support/travis_setup.php --source `pwd` --target ~/builds/ss --require silverstripe/cms
```
Comment (dhensby) [inline .travis.yml]: we can probably remove the `--require silverstipe/cms` (that should have been a hint that we need it as a composer dep ;) )

File: .travis.yml [hunk 2]
```diff
@@ -1,26 +1,33 @@
...
+  - cd ~/builds/ss
+  - composer install
```
Comment (dhensby) [inline .travis.yml]: This line is actually not needed as `travis_setup.php` does it for us, but I guess it's no harm

**Pull request merged**